package-lock.json

```diff
@@ -54,7 +54,7 @@
 			}
 		},
 		"apps/backend": {
-			"version": "1.8.0",
```

Comment:
You did not add any libraries also you did not remove any of them so package-lock.json file should have no changes